tracing-appender/Cargo.toml

```
@@ -2,12 +2,20 @@
 name = "tracing-appender"
 version = "0.1.0"
 authors = ["Zeki Sherif <zekshi@amazon.com>"]
```

Can you add `"Tokio Contributors <team@tokio.rs>"` here as well, please? Thanks!
```suggestion
authors = [
    "Zeki Sherif <zekshi@amazon.com>",
    "Tokio Contributors <team@tokio.rs>"
]
```